decanter_ai_sdk/client.py

```diff
@@ -520,6 +520,20 @@ def predict_ts(
         )
         return prediction
 
+    def stop_uploading(self, id: str):
```

Comment:
is the id logged for training and setup tasks?

Reply:
yes